README.md

```diff
@@ -11,3 +11,15 @@ This repository contains all the materials to be distributed through our worksho
 4. Add README.md containing all the resources and how to use that folder
 5. Submit a Pull Request!
 ```
+## Our Team 
```

Comment:
We need one more space above here.

Reply:
removing the leading |

Reply:
oops did I say space? I meant a `new line`. Sorry.